conda/environments/all_cuda-130_arch-x86_64.yaml

```diff
@@ -80,6 +80,7 @@ dependencies:
 - sphinxcontrib-openapi
 - sphinxcontrib-websupport
 - sysroot_linux-64==2.28
+- tbb-devel
```

Comment:
These came from running `pre-commit run --all-files` ... I did not add them.

That dependency was added in #371, but that PR wasn't updated after #366 was merged, so these CUDA 13 env files didn't exist yet.